### ISSUE TITLE
Audio input issue via USB-C/lightning 

### ISSUE THREAD

**Issue by cr0manty** [flutter-webrtc/flutter-webrtc#687]
On most devices, the microphone does not work through the usb-c / lightning port, after connecting, or if the microphone was before the start of broadcasting, then audio does not go. And through 3.5 jack the broadcast is going well

external input:
android - not work at all
iphone 12 with apple EarPods iOS 14.7.1 - works fine (audio input from headphone mic) 
<iPhone 12 - not work at all

**Comment by cloudwebrtc** [comment]
This is an interesting question, this should be an issue of audio manager or sound card management. Maybe we should develop an enhanced API to manage various audio devices, wired, wireless, Micro USB, type-c, 3.5mm, Bluetooth, It sounds like there are many things to do. xD

**Comment by cr0manty** [comment]
Yeah, I already start dig into this and it’s really not simple task, some base moves, like handle device notification and set audio on input via `setPreferredInput` Seems not work

**Comment by cr0manty** [comment]
@cloudwebrtc this pull request fix this issue on iOS https://github.com/flutter-webrtc/flutter-webrtc/pull/693
